dns-zone.tf

```diff
@@ -0,0 +1,12 @@
+data "azurerm_dns_zone" "play" {
+  name                = "us-hpccsystems-dev.azure.lnrsg.io"
+  resource_group_name = "app-dns-prod-eastus2"
```

Comment:
This should probably be a variable.

Reply:
> This should probably be a variable.
The 'data' block has been replaced with a "module" block and nothing in the module block is hardcoded.